scripts/Muon/fft_model.py

```diff
@@ -100,50 +102,28 @@ def FFTAlg(self,FFTInputs):
         group = mantid.AnalysisDataService.retrieve(self.runName)
         group.add(ws)
 
-    def makePhaseQuadTable(self,axis,instrument):
+    def makePhaseQuadTable(self,inputs):
         """
-        generates a phase table based on the detector setup
-        need to become an algorithm
+        generates a phase table from CalMuonDetectorPhases
         """
-        wsAlg=mantid.AlgorithmManager.create("CreateSimulationWorkspace")
-        wsAlg.initialize()
-        wsAlg.setChild(True)
-        wsAlg.setProperty("Instrument",instrument)
-        wsAlg.setProperty("BinParams","0,1,32")
-        wsAlg.setProperty("OutputWorkspace","__tmp__")
-        wsAlg.execute()
-        output=wsAlg.getProperty("OutputWorkspace").value
-
-        tableAlg=mantid.AlgorithmManager.create("CreateEmptyTableWorkspace")
-        tableAlg.initialize()
-        tableAlg.setChild(False)
-        tableAlg.setProperty("OutputWorkspace","PhaseTable")
-        tableAlg.execute()
-
-        phaseTable=mantid.AnalysisDataService.retrieve("PhaseTable")
-        phaseTable.addColumn("int","DetectorID")
-        phaseTable.addColumn("double","Phase")
-        phaseTable.addColumn("double","Asym")
-
-        for j in range(output.getNumberHistograms()):
-            det = output.getDetector(j).getPos()-output.getInstrument().getSample().getPos()
-            r=math.sqrt(det.X()**2+det.Y()**2+det.Z()**2)
-            if(axis=="x"):
-                phi=math.atan2(det.Z(),det.Y())
-                asym=math.sqrt(det.Z()**2+det.Y()**2)/r
-            elif(axis=="y"):
-                phi=math.atan2(det.X(),det.Z())
-                asym=math.sqrt(det.X()**2+det.Z()**2)/r
-            else: # z
-                phi=math.atan2(det.Y(),det.X())
-                asym=math.sqrt(det.Y()**2+det.X()**2)/r
-            phaseTable.addRow([j,asym,phi])
+        alg=mantid.AlgorithmManager.create("CalMuonDetectorPhases")
```

Comment:
The lack of whitespace makes this code quite dense. [PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations) recommends a space around the `=` operator for readability. Would it be possible to apply formatting to these changes to make them a bit more readable?

Reply:
will be able to do that when I get back in Jan